rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -4917,40 +5009,11 @@ mod test {
             GetWasmMetadataResponse as GetWasmMetadataResponsePb,
             MetadataSection as MetadataSectionPb,
         };
-        use libflate::gzip;
         use pretty_assertions::assert_eq;
 
-        // Adds section "icp:[public|private] $name$contents" to `wasm`, returning `wasm`'s new hash.
-        fn annotate_wasm_with_metadata(
-            wasm: &mut SnsWasm,
-            is_public: bool,
-            name: &str,
-            contents: Vec<u8>,
-        ) -> Vec<u8> {
-            use ic_wasm::{metadata, utils};
-
-            let kind = if is_public {
-                metadata::Kind::Public
-            } else {
-                metadata::Kind::Private
-            };
-
-            let mut wasm_module = utils::parse_wasm(&wasm.wasm, false).unwrap();
-            metadata::add_metadata(&mut wasm_module, kind, name, contents);
-
-            wasm.wasm = wasm_module.emit_wasm();
-
-            Sha256::hash(&wasm.wasm).to_vec()
-        }
-
         // Gzips a wasm, returning the hash of its compressed representation.
         fn gzip_wasm(wasm: &mut SnsWasm) -> Vec<u8> {
```

Comment:
This also seems wrong b/c it's returning a hash instead of a gzipped wasm.  Can you fix this one also?